x/bank/module.go

```diff
@@ -0,0 +1,69 @@
+package bank
+
+import (
+	"encoding/json"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// name of this module
+const ModuleName = "bank"
+
+// app module for bank
+type AppModule struct {
+	keeper        Keeper
+	accountKeeper auth.AccountKeeper
+}
+
+// NewAppModule creates a new AppModule object
+func NewAppModule(keeper Keeper, accountKeeper auth.AccountKeeper) AppModule {
```

Comment:
can we pass keys in instead and create the keepers in the modules?

Reply:
Due to the interdependence of the keepers on each other that would make things much uglier at  the `app.go` level. I think the high level wiring of the keepers together should stay as is